README.md

```diff
@@ -1 +1,120 @@
-# CodersCamp2021.Project.Fullstack
\ No newline at end of file
+
+# CodersCamp 2021 - Projekt Node.js
+
+<div align="center">
+
+[![Demo](https://img.shields.io/badge/-demo-green?logo=github)](https://coderscamp2021-hk-fullstack.herokuapp.com/)
+[![Figma](https://img.shields.io/badge/-brainstorming+wireframing-blueviolet?logo=figma)](https://www.figma.com/file/51gzaCTbilz6X9r0PZxmhX/Project.Fullstack?node-id=0%3A1)
+</div>
+
+<br/>
+
+<p>
+🍲🍝🍜 <strong>BierzCoChcesz</strong> 🍲🍝🍜 projekt skierowany do ludzi, którzy nie lubią gdy narzuca im się wybory jedzeniowe a jednocześnie zwracają uwagę na wartości odżywcze swoich posiłków.
+Aplikacja pozwala na skomponowanie przez użytkownika diety na cały dzień składającej się z dań z ulubionych restauracji. W praktyce to system partnerski zrzeszający najlepsze restauracje.</p>
+
+<br/>
+<br/>
+
+## 🖥️ Użyte technologie
+<br/>
+
+### Główne:
+ - [Docker](https://docs.docker.com/get-docker/)
+ - [MongoDB](https://www.mongodb.com/)
+ - [Mongoose](https://mongoosejs.com/)
+ - [NestJS](https://nestjs.com/)
+ - [Express](https://expressjs.com/)
+ - [Nodemailer](https://nodemailer.com/)
+ - [Swagger](https://swagger.io/tools/swaggerhub/?&utm_medium=ppcg&utm_source=aw&utm_term=swagger&utm_content=511173019632&utm_campaign=SEM_SwaggerHub_PR_EMEA_ENG_EXT_Prospecting&awsearchcpc=1&gclid=CjwKCAjwxOCRBhA8EiwA0X8hiwxh7NHt37o-DaApRMBtiZ25QonxHVTKTew0Pa2SYAC6nDVzJnYD3xoCejMQAvD_BwE&gclsrc=aw.ds)
+ - [Figma](https://www.figma.com/files/recent?fuid=1047244325467558592)
+
+<br/>
+
+### Pomocnicze:
+- TypeSript
+- Yarn
+- Jest
+- ESLint
+- VS Code
+- Git
+- GitHub
+- Figma 
+- PassportJWT
+- REST Client
+<br/>
+<br/>
+
+## ⚙️ Instalacja projektu
+Do przygotowania środowiska dev użyliśmy m.in. [Dockera](https://docs.docker.com/get-docker/), więc zakładamy, że jest on wcześniej zainstalowany i skonfigurowany. 
+<br/>
+<br/>
+
+#### Instalacja i urochomienie projektu z wykorzystaniem `yarn`
+
+```bash
+yarn install
+docker-compose up
+```
+
+Obserowanie zmian w środowisku dev:
+```bash
+yarn dev
+```
+
+Wygenerowanie plików produkcyjnych:
+```bash
+yarn build
+```
+<br/>
+
+#### Pozostałe komendy
+
+Sprawdzenie projektu za pomocą [ESLint](https://eslint.org/):
+```bash
+yarn lint
+```
+Uruchomienie testów jednostkowych:
+```bash
+yarn test:unit
+```
```

Comment:
Chyba nie mamy testów jednostkowych

Reply:
racja, pisałam to co było w package.json